### PR TITLE
Fix errors

### LINKED ISSUE #1937: Emojis panel appears in wrong place
### Problem

On desktop in Brave, I clicked on the emojis icon and the panel appeared way up in the upper-right:

Screenshot 2024-04-17 at 10 28 49 AM

### Solution

The panel should appear next to the emoji button, check Slack or Discord for how this should be done.

### LINKED ISSUE #1940: Mobile UI issue switching between chats
### Problem

I encountered this on mobile (Android/GrapheneOS) when switching from the general chatroom to a DM:

Screenshot_20240417-1953261

Also happens when going from the DM back to the general chatroom.

### Solution

Fix.

### PR DESCRIPTION
**Pull request by Silver-IT**
### Summary of Changes

- Closes #1937 
- Closes #1940

### CODE REVIEW

File: frontend/views/pages/GroupChat.vue [hunk 1]
```diff
@@ -82,16 +82,23 @@ page(pageTestName='groupChat' :miniHeader='isDirectMessage()')
...
+        @redirect='toggle'
       )
 
-      chat-members(:title='L("Direct Messages")' action='addDirectMessage')
+      chat-members(
+        action='addDirectMessage'
+        :title='L("Direct Messages")'
+        @new='toggle'
+        @redirect='toggle'
+      )
```
Comment (taoeffect) [inline frontend/views/pages/GroupChat.vue:101]: What is this `toggle`? Where is it defined?

**Review by taoeffect — COMMENTED** [review]
Nice! A quick question:

File: frontend/views/pages/GroupChat.vue (hunk 1 above)
Comment (Silver-IT) [reply frontend/views/pages/GroupChat.vue:101]: Here, in `views/components/Page.vue`.
```
-     slot(name='sidebar')
+     slot(name='sidebar' :toggle='toggleMenuIfTouch')
```

File: frontend/common/translations.js [hunk 2]
```diff
@@ -133,13 +133,14 @@ export default function L (
...
   let url = `/app/dashboard?modal=UserSettingsModal&tab=application-logs&errorMsg=${encodeURI(error.message)}`
+  const target = !sbp('state/vuex/state').loggedIn ? 'target="_blank"' : ''
   if (!sbp('state/vuex/state').loggedIn) {
     url = 'https://github.com/okTurtles/group-income/issues'
   }
   return {
     reportError: L('"{errorMsg}". You can {a_}report the error{_a}.', {
       errorMsg: error.message,
-      'a_': `<a class="link" target="_blank" href="${url}">`,
+      'a_': `<a class="link" ${target} href="${url}">`,
```
Comment (taoeffect) [inline frontend/common/translations.js]: Per [this comment](https://github.com/okTurtles/group-income/pull/1965#discussion_r1585236100) - please if you could revert these changes 🙏

**Review by taoeffect — CHANGES_REQUESTED** [review]
Nice! Tested and seems to work well 👍 

Just one change request

**Review by taoeffect — APPROVED** [review]
This seems to work well, nicely done! 👍

**Pull request merged**